main/background.ts

```diff
@@ -46,13 +46,17 @@ function metric_string_parse(item): number | null {
   if (!item) return null;
   return +item.metrics[0].value;
 }
-ipcMain.on("metrics", async (event) => {
+
+// better dev quality, temp solution
+async function getMetrics(): Promise<Metrics> {
+  console.log("DEBUG: getMetrics start");
   const res = await fetch("http://testnet-3.arweave.net:1984/metrics");
   const data = await res.text();
```

Comment:
unrelated to the PR, but we are perhaps missing try/catch here?

Reply:
For context of this PR no. Because I want that promise should be error, so FE will also receive error, so I can see it in browser console

Reply:
you may end up crashing nodejs if it's not caught anywhere, in a single threaded system it would be fatal error. It's not a big deal for now, I mainly thinking it makes sense to give better error messages. Uncaught errors in promises are quite hard to read and understand.